descriptors/ModuleDescriptor-template.json

```diff
@@ -634,6 +634,37 @@
           "delay": "1"
         }
       ]
+    },
```

Comment:
lets move this declaration after custom-fields before system interfaces _tenant, _timer